internal/features/stacks/state/stacks_store.go

```diff
@@ -17,6 +18,11 @@ type StackStore struct {
 	db        *memdb.MemDB
 	tableName string
 	logger    *log.Logger
+
+	// MaxStackNesting represents how many nesting levels we'd attempt
+	// to parse provider requirements before returning error.
+	MaxStackNesting int
+	changeStore     *globalState.ChangeStore
```

Comment:
I didn't add this, must've come while resolving a conflict during rebasing – is this safe to remove? cc @jpogran